cmd/system-probe/modules/eventmonitor.go

```diff
@@ -91,5 +94,13 @@ func createEventMonitorModule(_ *sysconfigtypes.Config, deps module.FactoryDepen
 		}
 	}
 
+	gpucfg := gpuconfig.New()
+	if gpucfg.Enabled {
+		err := createGPUProcessEventConsumer(evm)
```

Comment:
So I'm not 100% sure, and I still have a TODO list item to refactor this, but don't you need to call `evm.RegisterEventConsumer(` here, like other consumers ?

Reply:
That's included in the `createGPUProcessEventConsumer` function,  more specifically [here](https://github.com/DataDog/datadog-agent/blob/guillermo.julian/gpu-event-consumer/pkg/eventmonitor/consumers/process.go#L98) in the generic method for process-data consumers.

Reply:
That's a different consumer, for forwarding data to process-agent I think. This one is the one I introduced in [this PR](https://github.com/DataDog/datadog-agent/pull/30559) to have a generic consumer that can be used to replace the process monitor, which turned out to be only for USM use.